website/docs/guides/fusion-package-compat.md

```diff
@@ -0,0 +1,168 @@
+---
+title: "Fusion package compatibility guide"
+id: "fusion-package-compat"
+description: "Learn how to upgrade your packages to be compatible with the dbt Fusion engine."
+intro_text: "Learn how to upgrade your packages to be compatible with the dbt Fusion engine."
+hoverSnippet: "Learn how to upgrade your packages to be compatible with the dbt Fusion engine."
+# time_to_complete: '30 minutes' commenting out until we test
+icon: 'zap'
+hide_table_of_contents: true
+tags: ['dbt Fusion engine']
+level: 'Advanced'
+---
+
+## Introduction
+
+Thank you for being part of the [dbt package community](https://getdbt.slack.com/archives/CU4MRJ7QB) and maintaining [dbt packages](https://hub.getdbt.com/)! Your work makes dbt’s ecosystem possible and helps thousands of teams reuse trusted models and macros to build faster, more reliable analytics.
+
+This guide will help you upgrade your dbt packages to be [<Constant name="fusion" />](/docs/fusion)-compatible. A <Constant name="fusion" />-compatible package:
+- Supports [<Constant name="fusion_engine" />](/docs/fusion) version `2.0.0`, 
+- Uses the [`require-dbt-version` config](/reference/project-configs/require-dbt-version) to signal compatibility in the dbt package hub,
+- aligns with the latest JSON schema introduced in dbt <Constant name="core"/> v1.10.0.
+
+In this guide, we'll go over:
+
+- Updating your package to be compatible with <Constant name="fusion"/>,
+- Testing your package with <Constant name="fusion"/>,
+- Updating the `require-dbt-version` config to include `2.0.0`,
+- Updating your README to note that the package is compatible with <Constant name="fusion"/>.
+
+### Who is this for?
```

Comment:
Do we use a "Who is this for" section in other guides? I wonder if we should instead add a persona in the metadata for the package maintainer so they can sort by it?